spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/IcebergSource.java

```diff
@@ -132,6 +168,14 @@ public String extractCatalog(CaseInsensitiveStringMap options) {
     return catalogAndIdentifier(options).catalog().name();
   }
 
+  private static Long getPropertyAsLong(CaseInsensitiveStringMap options, String property) {
```

Comment:
How about `propertyAsLong` instead? There isn't much value in `get` here.

Reply:
Renamed.